Tests/Find-ImageName.Tests.ps1

```diff
@@ -1,13 +1,101 @@
 Import-Module -Force $PSScriptRoot/../Docker.Build.psm1
 
+function Get-TempPath {
+    if ($IsWindows) {
+        [system.io.path]::GetTempPath()
+    } elseif ($IsLinux) {
+        '/tmp'
+    }
+}
+
+function New-RandomFolder {
+    param (
+        [int]$FolderLength = 8
+    )
+    do {
+        $randomString = ( -join ((0x30..0x39) + ( 0x41..0x5A) + ( 0x61..0x7A) | Get-Random -Count $FolderLength | % {[char]$_}) )
```

Comment:
I would suggest using GUID to come up with a globally unique folder name instead, see https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/new-guid?view=powershell-6

Reply:
Done!